agent/bench-scripts/pbench-uperf

```diff
@@ -688,6 +688,7 @@ done
 echo "$script_path/postprocess/generate-benchmark-summary \"$benchmark\" \"$orig_cmd\" \"$benchmark_run_dir\"" >"$benchmark_run_dir/generate-benchmark-summary.cmd"
 chmod +x "$benchmark_run_dir/generate-benchmark-summary.cmd"
 $script_path/postprocess/generate-benchmark-summary "$benchmark" "$orig_cmd" "$benchmark_run_dir"
+pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir
```

Comment:
Add "`end`" argument, and invoke it with the "`beg`" argument between lines 475 and 476.